source/distributed/grid_refinement.cc

```diff
@@ -241,15 +241,15 @@ namespace internal
                                            global_min_and_max.second};
             adjust_interesting_range(interesting_range);
 
-            const unsigned int master_mpi_rank = 0;
+            const unsigned int parent_mpi_rank = 0;
```

Comment:
calling rank 0 in MPI the "parent" is confusing to me. In the MPI world people often use "root" instead.

Reply:
Yes, I think "root" is better.

Reply:
Thanks for the suggestion. I've now used "root" everywhere when referring to the MPI rank.